config/routes.d/staging.horse.json

```diff
@@ -4,7 +4,8 @@
       "^/docs/private-cloud/release-notes/": "developer.rackspace.com/docs-singlepage.html",
       "^/docs/private-cloud/rpc/eng-handbook/": "developer.rackspace.com/docs-singlepage.html",
       "^/docs/private-cloud/": "developer.rackspace.com/user-guide.html",
-      "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html"
+      "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html",
+      "^/docs/billing-apiguide/": "developer.rackspace.com/docs-singlepage.html"
```

Comment:
Also, same note applies here as on one of the other PRs about the URL. As long as it's the same there and here, it won't matter what it is, but just something for you to think about.